examples/nivisa_dmm_measurement/teststand_fixture.py

```diff
@@ -37,13 +36,16 @@ def update_pin_map(pin_map_path: str, sequence_context: Any) -> None:
     teststand_support.set_active_pin_map_id(pin_map_id)
 
 
-def create_nivisa_dmm_sessions(sequence_context: Any) -> None:
+def create_nivisa_dmm_sessions(sequence_context: Any, use_simulation: bool) -> None:
```

Comment:
If we are going to move `use_simulation` to the TestStand file globals:
- Do it for all examples in a separate PR
- Update README.md

Reply:
I agree with that approach for all examples that support simulation. Add a new File Global as sibling to the MeasurementLink container (not within it).

Reply:
Resolving the conversation now, but the change will be taken up as part of work item 2473562.